portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,184 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+#box {
+    max-width: 2000px;
+    margin:auto;
+    background-color:rgb(253, 234, 240);
 }
 
-#greeting-container {
-  margin-top: 20px;
+#project-showcase {
+    max-width: 2000px;
+    margin:auto;
+    background-color:rgb(253, 234, 240);
 }
+
+.column {
+    margin-left:62px;
+    text-align:center;
+}
+
+h1 {
+    font-family: 'Lucida Console', Monaco, monospace;
+    text-align: center;
+    margin-top:70px;
+}
+
+h2 {
+    font-family: 'Courier New', Courier, monospace;
```

Comment:
Sorry... I'm a bit of a typography nerd so I have to quibble with the juxtaposition of Courier and Lucida Console. I'm not convinced they go well together. This however is completely a matter of a taste so you should freely ignore this.

Reply:
I actually value this comment a lot, I was struggling with which ones to choose. I do think I like both separately, but I agree that they have different tones. I did some more digging and pairing. What do you think about Courier New with DM Mono (a Google font)?

Reply:
I haven't used DM Mono so I don't have a good sense of what it looks like. Here are some articles about type that I like:

* [https://betterwebtype.com/articles/2018/10/15/rhythm-in-web-typography/](https://betterwebtype.com/articles/2018/10/15/rhythm-in-web-typography/)
* [https://www.robinrendle.com/essays/the-new-web-typography](https://www.robinrendle.com/essays/the-new-web-typography)

I've been a Tschichold fan for a while. I name my phones after designers and use to have one called *Tschichold*.

Reply:
Will definitely take a closer look at those articles, thank you for your opinion!